apps/widget/cypress/plugins/index.ts

```diff
@@ -99,6 +112,12 @@ module.exports = (on, config) => {
       };
     },
 
+    async awaitRunningJobs({ templateId, organizationId }: { templateId: string; organizationId: string }) {
+      await awaitRunningJobs({ apiKey: config.env.API_URL, templateId, organizationId });
+
+      return true;
+    },
```

Comment:
you don't need to have this wrapper function, instead assign the default value to the `awaitRunningJobs:132`, like 
```
async function awaitRunningJobs({
  apiKey = config.env.API_URL,
...
```

Reply:
Created this one to use it in `notifications-list.spec.ts` in order to try to solve the flaky tests in the CI. :(

Reply:
sorry what I meant is that you don't have to call `awaitRunningJobs` in line 116... instead you can do:
```
 const session = new UserSession(serverUrl);
  await session.awaitRunningJobs(templateId, undefined, 0, organizationId);
```
like you don't need to have that additional function `awaitRunningJobs`